Autopilot/TelemetryManager/Telemetry_Manager.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Telemetry_Manager.h
+ * @author Jingting Liu
+ * @date January 14th, 2021
+ */
+
+
+#ifndef _TELEMETRY_STATE_MACHINE_H_
+#define _TELEMETRY_STATE_MACHINE_H_
+
+
+typedef enum TM_STATE_e {
+    INIT,
+    WAIT_FOR_GROUND_DATA,
+    CONNECT_WITH_PATH_MANAGER,
+    REGULAR_REPORT_READY,
+    ERR_REPORT_READY,
+    SEND_ERR_REPORT,
+    SEND_REGULAR_REPORT,
+} TM_STATE_e;
+
+void telemetry_manager(void);
```

Comment:
We have a standard way of doing state machines. It's not too much better than this, but I'd like all our state machines to follow the same structure. Take a look at AutoPilot/AttitudeManager to see how it's done.